src/modules/layerManager/components/LayerManager.js

```diff
@@ -178,9 +178,9 @@ export class LayerManager extends MvuElement {
 			<style>
 				${css}
 			</style>
-			<div class="layermanager overflow-container">
+			<div class="layermanager overflow-container draggable">
```

Comment:
should not be draggable, i guess...
this makes the hole container draggable, which includes the action buttons (['expand all'], ['remove all'])